### PR TITLE
Add some tests for CodeAnalysisExtensions

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
FYI @kichalla

File: src/Microsoft.AspNetCore.Mvc.Analyzers/CodeAnalysisExtensions.cs [hunk 1]
```diff
@@ -11,6 +11,14 @@ internal static class CodeAnalysisExtensions
     {
         public static bool HasAttribute(this ITypeSymbol typeSymbol, ITypeSymbol attribute, bool inherit)
         {
+            Debug.Assert(typeSymbol != null);
```
Comment (kichalla) [inline src/Microsoft.AspNetCore.Mvc.Analyzers/CodeAnalysisExtensions.cs]: Are these extensions common enough to go into a common repo that could be used by analyzers written in other repos?

**Comment by pranavkm** [comment]
@dotnet-bot test this please

File: src/Microsoft.AspNetCore.Mvc.Analyzers/CodeAnalysisExtensions.cs (hunk 1 above)
Comment (pranavkm) [reply src/Microsoft.AspNetCore.Mvc.Analyzers/CodeAnalysisExtensions.cs]: We can consider this if there's an ask. Most of the logging analyzers look at strings so they wouldn't benefit from this.

**Pull request merged**